src/main/java/ljv/LJV.java

```diff
@@ -24,253 +24,236 @@
 import java.util.*;
 
 class LJV {
-    private final IdentityHashMap<Object, String> objectsId = new IdentityHashMap<>();
+    private final Map<Object, String> classAttributeMap = new HashMap<>();
+    private final Map<Object, String> fieldAttributeMap = new HashMap<>();
+    private final Set<Object> pretendPrimitiveSet = new HashSet<>();
+    private final Set<Object> ignoreSet = new HashSet<>();
 
-    private String dotName(Object obj) {
-        return obj == null ? "NULL" : objectsId.computeIfAbsent(obj, s -> "n" + (objectsId.size() + 1));
+    private enum Directions {
+        BT, LR, TB, RL;
+    }
+    private Directions direction = Directions.TB;
+
+    public LJV setDirection(String directionStr) {
```

Comment:
It's better to make `Directions` a public enum (`Direction` must be a better name) and use `setDirection(Direction direction)`